bomeba0/energy/ff.py

```diff
@@ -1,12 +1,251 @@
 """
-Draft of a force-field. At this point only a Lennard Jones term is implemented.
+Draft of a force-field.
+At this point only a Lennard Jones and solvation terms are implemented.
+
+Functions assign_atom_types(), assign_params(), get_neighbors() and, 
+get_sasa() adapted from
+https://github.com/BIOS-IMASL/Azahar/blob/master/Azahar/energy.py
 """
+
+import openbabel as ob
+import numpy as np
+from dictionaries import *
```

Comment:
Why you need this package? You should avoid importing everything.

Reply:
Here go the details:

`openbabel` is used by `assign_atom_types()`, we can change this but it is necessary to reformulate the entire function.

`numpy` is used in `get_sasa()`: the constant `np.pi` is used but I didn't see that this constant is in the file `utils/constant.py`, so that can be changed for that constant. `np.zeros` is used in `nb_matrix = np.zeros((len(points_scaled), len(neighbors)))` and `np.sum` is used in `buried = np.sum(np.sum(d_matrix < nb_matrix + probe, axis=1) > 0)`. For these last two, changing the `import numpy as np` to `from numpy import sum, zeros` could it be enough?

Lastly, I ran some tests, and `from dictionaries import *` is an error that remained from when I started to write the functions, so that line can be deleted.